INSTALL.md

```diff
@@ -1,25 +1,37 @@
 # Installing
 
+## Apple macOS
+Install via [Homebrew Cask](https://brew.sh/) or download a pre-built app bundle for macOS (10.13+) from [Releases](https://github.com/borgbase/vorta/releases).
+```
+$ brew cask install vorta
+```
 
-## Install from Source
-The generic way is to install it as Python package using [PIP](https://pip.readthedocs.io/en/stable/installing/). First [install](https://borgbackup.readthedocs.io/en/stable/installation.html) Borg using the package of your distribution or via PyPI. The latter needs some additional [source packages](https://borgbackup.readthedocs.io/en/stable/installation.html#dependencies). Then install Vorta from PyPI. Your local Python version should be >= 3.6.
+## Linux
+
+### Flatpak
+
+<a href='https://flathub.org/apps/details/com.borgbase.Vorta'><img width='240' alt='Download on Flathub' src='https://flathub.org/assets/badges/flathub-badge-en.png'/></a>
+
+Settings can be transfered by copying ~/.local/share/Vorta/settings.db to ~/.var/app/com.borgbase.Vorta/data/Vorta/.
+
+### Distribution Packages
+If you added Vorta to a Linux distribution, open an issue or PR and it will be added here.
```

Comment:
this should not be the first sentence here, as it addresses only a very small part of the readers (who happen to be vorta package maintainers).

see above for some more suitable content.

Reply:
What do you mean with "see above"?

Reply:
https://github.com/borgbase/vorta/pull/310#discussion_r302552864